### DISCUSSION TITLE
[求解答]肾上腺素威力真的有那么大吗？

### DISCUSSION THREAD

**Discussion by L502567** [share121/inter-knot#1949] [其他]
https://v.kuaishou.com/QqER1X 无法想象肾上腺素到底有多强 该作品在快手被播放过684.3万次，点击链接，打开【快手极速版】直接观看

**Comment by Phaethon-0019** [comment]
视频里有些情节肯定夸张了，但肾上腺素确实NB，Apex里动力小子扎一针跑飞快；之前有个从缅北跑四小时跑回国的重庆大哥就是靠肾上腺素跑回来的，你尽管发力，剩下的交给肾上腺素
